.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

Comment:
Make sure that ignoring the `logs` directory is intentional and aligns with your project's requirements. If log files are needed for debugging or sharing, consider adjusting this rule.